### ISSUE TITLE
sbt-assembly fails due to repeated and conflicting jar references

### ISSUE THREAD

**Issue by searler** [scodec/scodec-stream#2]
Note that both _scodec-bits_2.10-1.0.5.jar_ and _scodec-bits_2.10-1.0.4.jar_ are referenced

A project that only references scodec-core does not fail.

```
[info] Including from cache: scodec-bits_2.10-1.0.5.jar
[info] Including from cache: scodec-stream_2.10-0.7.0.jar
[info] Including from cache: scalaz-stream_2.10-0.6a.jar
[info] Including from cache: scalaz-effect_2.10-7.1.0.jar
[info] Including from cache: scodec-core_2.10-1.7.0.jar
[info] Including from cache: scodec-bits_2.10-1.0.4.jar
[info] Including from cache: shapeless_2.10.4-2.1.0.jar
[info] Including from cache: quasiquotes_2.10-2.0.0.jar
[info] Including from cache: scala-reflect.jar
[info] Including from cache: scala-library-2.10.5.jar
[info] Including from cache: scalaz-core_2.10-7.1.0.jar
[info] Including from cache: scalaz-concurrent_2.10-7.1.0.jar
[info] Checking every *.class/*.jar file's SHA-1.
[info] Merging files...
[warn] Merging 'META-INF/MANIFEST.MF' with strategy 'discard'
[error] 74 errors were encountered during merge
java.lang.RuntimeException: deduplicate: different file contents found in the following:
/home/rsearle/.ivy2/cache/org.scodec/scodec-core_2.10/bundles/scodec-core_2.10-1.7.0.jar:scodec/bits/Bases$Alphabets$Base64$.class
/home/rsearle/.ivy2/cache/org.scodec/scodec-bits_2.10/bundles/scodec-bits_2.10-1.0.5.jar:scodec/bits/Bases$Alphabets$Base64$.class
/home/rsearle/.ivy2/cache/org.typelevel/scodec-bits_2.10/bundles/scodec-bits_2.10-1.0.4.jar:scodec/bits/Bases$Alphabets$Base64$.class
```

build.sbt
```
name := "scodec streaming assembly"

version := "0.1"

scalaVersion := "2.10.5"

resolvers += "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases"

libraryDependencies ++= Seq(
   "org.scodec" %% "scodec-stream" % "0.7.0"
)
```
project/build.properties 
```
sbt.version=0.13.7
```

project/assembly.sbt 
```
addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.13.0")
```

**Comment by mpilquist** [comment]
Looks like scodec-core is erroneously including the scodec-bits class files, most likely due to a bad sbt-osgi Export-Package setting.

**Comment by mpilquist** [comment]
Fixed in https://github.com/scodec/scodec/issues/49. I pushed an scodec-core 1.7.1-SNAPSHOT with the fix. I hope to have scodec-core 1.7.1 released soon -- probably in the next day or two.

**Comment by mpilquist** [comment]
I just released scodec-stream 0.7.1 which depends on scodec-core 1.7.1. That should fix this issue.
